pkg/commands/process/settings/rules/javascript/express/https_protocol_missing.yml

```diff
@@ -0,0 +1,32 @@
+patterns:
+  - pattern: |
+      https.createServer()
+    filters:
+trigger_rule_on_presence_of: javascript_express_https_protocol_missing_http_server
+auxiliary:
+  - id: javascript_express_https_protocol_missing_http_server
+    patterns:
+      - $<_>.createServer()
+languages:
+  - javascript
+trigger: absence
+severity:
+  default: "medium"
+metadata:
+  description: "Security misconfiguration detected."
+  remediation_message: |
+    ## Description
+
+    Helmet can help protect your app from some well-known web vulnerabilities by setting HTTP headers appropriately.
```

Comment:
Description seems wrong here?

Reply:
🙈

Reply:
Thanks for the catch @gotbadger - updated 👍

Reply:
Nice catch @gotbadger